app/renderer/public/index.html

```diff
@@ -4,6 +4,7 @@
     <meta charset="UTF-8" />
     <link rel="icon" href="%PUBLIC_URL%/favicon.ico" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <meta http-equiv="Content-Security-Policy" content="default-src 'self' data: 'unsafe-inline'">
```

Comment:
Just gets rid of a warning and allows us to be a bit more secure/specific with loaded scripts